library/options.inc.php

```diff
@@ -373,7 +373,7 @@ function generate_form_field($frow, $currvalue) {
       " value='" . $formDate . "' $disabled />";
     if (!$disabled) {
        $selector = "#form_{$field_id}";
-       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false, format:'" . $DateFormat . "'});";
+       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false,  format:'" . $DateFormat . "'});";
```

Comment:
I fixed these in Updating Docs pull request, so I thought,fixing it in this pull request may create problems in future. 
Is it not the case?